### PR TITLE
Update ChangeLog

### PR DESCRIPTION
**Pull request by rohandubal**
*Description of changes:*

Add CHANGELOG for `2.7.0`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,20 @@ The AWS AppSync SDK for iOS enables you to access your AWS AppSync backend and p
...
 ## 2.7.0
 
+### New Features
+
+* Added support for Delta Sync Feature
+    Delta Sync allows you to perform automatic synchronization with an AWS AppSync GraphQL server. The client will perform reconnection, exponential backoff, and retries when network errors take place for simplified data replication to devices. For more details, please refer [documentation.](https://aws-amplify.github.io/docs/ios/api)
+
+### Bug Fixes
+
+* Fix issue where if a time out error occurs, the callback would not be given back to developer. [See issue #91](https://github.com/awslabs/aws-mobile-appsync-sdk-ios/issues/91)
```
Comment (cbommas) [inline CHANGELOG.md]: Change to "Fixed issue where if a timeout error occurred due to lack of network availability, the callback would not be given back to the developer"

**Comment by shannon-hager-skookum** [comment]
``` Added support for Delta Sync Feature
Delta Sync allows you to perform automatic synchronization with an AWS AppSync GraphQL server. The client will perform reconnection, exponential backoff, and retries when network errors take place for simplified data replication to devices. For more details, please refer [documentation.](https://aws-amplify.github.io/docs/ios/api)```

I'm not sure that is the correct URL. There are no matches on that page for any of the applicable keywords: "delta sync", "delta", "synchronization", "reconnection", "exponential", "backoff", "retries", "retry", "simplified", "replication", "replicate"

**Comment by undefobj** [comment]
@shannon-hager-skookum the deployment and documentation updates are still ongoing for backend service changes. You will see the updated page in the URL later this evening.

**Comment by shannon-hager-skookum** [comment]
@undefobj Thanks. I see it now at https://aws-amplify.github.io/docs/ios/api#delta-sync

**Pull request merged**